### DISCUSSION TITLE
StreetComplete Rank

### DISCUSSION THREAD

**Discussion by Suburbanno** [streetcomplete/StreetComplete#2424] [General]
Hey, I hope you are well!

I would like to know what this rank is based on and if there is a website to check the classification or if there is a query API

Hug!

![image](https://user-images.githubusercontent.com/13632826/103174602-4655c680-4842-11eb-8de5-e5f5d1756cc9.png)

**Comment by matkoniecz** [comment]
Congratulations and thanks for mapping!

> I would like to know what this rank is based on

Number of uploaded changes made with StreetComplete (edit counter is visible next to star in the main view)

> and if there is a website to check the classification or if there is a query API

There is no website or public api. Though https://github.com/streetcomplete/StreetComplete/issues/2334#issuecomment-737288879 has a bit of info about distances at top of classification.

**Reply by Suburbanno** [reply]
thanks guy!

**Reply by westnordost** [reply]
https://github.com/streetcomplete/StreetComplete/issues/2334#issuecomment-737288879 was for global ranks though.
